apps/sms/services/js/conversation/conversation_service.js

```diff
@@ -0,0 +1,89 @@
+/*global ServiceMixin */
+'use strict';
+
+importScripts('../service_mixin.js');
+
+(function() {
+  var service;
+
+  var ConversationService = {
```

Comment:
I was not sure; but now that I think more of it I think I can try to include it in service mixin